cinema/views.py

```diff
@@ -48,6 +89,37 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
     queryset = MovieSession.objects.all()
     serializer_class = MovieSessionSerializer
 
+    @staticmethod
+    def param_to_id(qs):
+        return [int(str_id) for str_id in qs.split(",")]
```

Comment:
make ur code dry u have already implemented this method, if u need to use this method in few classes u can write a mixin